app/models/session.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 class Session < RedisStore
-  NAMESPACE = REDIS_CONFIG['session_store']['namespace']
-  REDIS_STORE = Redis::Namespace.new(NAMESPACE, redis: Redis.current)
-  DEFAULT_TTL = REDIS_CONFIG['session_store']['each_ttl']
+  redis_store REDIS_CONFIG['session_store']['namespace']
+  default_ttl REDIS_CONFIG['session_store']['each_ttl']
```

Comment:
Huge fan of doing it this way vs Constants. That was a total hack I put together quickly. 👍